tests/test_wave_01.py

```diff
@@ -143,12 +144,13 @@ def test_moves_movie_from_watchlist_to_watched():
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
     
-    raise Exception("Test needs to be completed.")
+    assert updated_data["watched"][1]["title"] == movie_to_watch["title"]
```

Comment:
Another way to test this could be to use `in` statements:
```python
assert movie_to_watch in updated_data["watched"]
```
That strategy could be used for verifying each of the two movies we'd expect to find in the watched list as well as the one in the watchlist list.